[Bug]: 1.11.0: No matching version found for @lit-labs/observers@^3.0.0OBSERVERS

### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Other

### It affects the following packages

Components

### Which version of Lyne Components are you using

1.11.0

### Operating system

any

### Browser / Browser version

not relevant

### Input mode

None

### Additional settings

_No response_

### What happened?

I'm trying to update to Lyne 1.11.0 (thanks for the fixes!). Unfortunately something seems to be off with the dependencies. When installing them, I get the error "`ETARGET: No matching version found for @lit-labs/observers@^3.0.0OBSERVERS.`" Yarn will let me choose an existing version, npm will just throw an error and exit.

### Steps to reproduce the issue

1. Visit https://stackblitz.com/edit/angular-at-8hg8bk?file=package.json
2. Run `npm install` or `yarn install`
3. The error should appear


### Relevant log output

```shell
❯ npm i
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @lit-labs/observers@^3.0.0OBSERVERS.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in: /home/.npm/_logs/2024-09-20T06_45_39_932Z-debug-0.log
```

--- a/tools/vite/package-json-template.ts
+++ b/tools/vite/package-json-template.ts
@@ -26,21 +26,21 @@ export function packageJsonTemplate(
       const rootPackageJsonPath = './package.json';
       const packageJsonTemplatePath = options.templatePath ?? './package.json';
       const rootPackageJson = JSON.parse(readFileSync(new URL(rootPackageJsonPath, root), 'utf8'));
-      const litMajorVersion = +rootPackageJson.dependencies.lit.match(/\d+/);
-      const litObserversMajorVersion =
-        +rootPackageJson.devDependencies['@lit-labs/observers'].match(/\d+/);
+      const litVersion = rootPackageJson.dependencies.lit.match(/\d+\.\d+\.\d+/);
+      const litObserversVersion =
+        rootPackageJson.devDependencies['@lit-labs/observers'].match(/\d+\.\d+\.\d+/);
       const reactMajorVersion = +rootPackageJson.devDependencies.react.match(/\d+/);
-      const litReactMajorVersion = +rootPackageJson.devDependencies['@lit/react'].match(/\d+/);
+      const litReactVersion = rootPackageJson.devDependencies['@lit/react'].match(/\d+\.\d+\.\d+/);
       const packageJsonTemplate = readFileSync(
         join(viteConfig.root, packageJsonTemplatePath),
         'utf8',
       );
       const packageJsonContent = packageJsonTemplate
         .replaceAll('0.0.0-PLACEHOLDER', rootPackageJson.version)
-        .replaceAll('0.0.0-LITREACT', `^${litReactMajorVersion}.0.0`)
+        .replaceAll('0.0.0-LITOBSERVERS', `^${litObserversVersion}`)
+        .replaceAll('0.0.0-LITREACT', `^${litReactVersion}`)
         .replaceAll('0.0.0-REACT', `^${reactMajorVersion}.0.0`)
-        .replaceAll('0.0.0-LIT', `^${litMajorVersion}.0.0`)
-        .replaceAll('0.0.0-LITOBSERVERS', `^${litObserversMajorVersion}.0.0`);
+        .replaceAll('0.0.0-LIT', `^${litVersion}`);
       const packageJson = JSON.parse(packageJsonContent);
       for (const key of ['author', 'license', 'repository', 'bugs']) {
         packageJson[key] = rootPackageJson[key];

